src/components/Footer/Footer.tsx

```diff
@@ -14,7 +14,12 @@ export function Footer() {
                 target="_blank"
                 href="https://www.accessibility-services.co.uk/certificates/national-institute-for-health-and-care-research/"
               >
-                <Image src="/assets/logos/shaw-trust.png" height={57} width={189} alt="Shaw Trust Logo" />
+                <Image
+                  src="/assets/logos/shaw-trust.png"
+                  height={57}
+                  width={189}
+                  alt="Shaw Trust Logo (Opens in a new window)"
```

Comment:
@paddyduff31 alt is used to describe the image, but we want to describe the context of the link to the user. Instead we should be adding and aria-label to the link e.g. aria-label="shaw trust accessibility website (opens in a new window)"

Reply:
@kevinofficer ahhh fair point!  Should probably be on the `<a>` tag too